salt/modules/virt.py

```diff
@@ -1696,19 +1697,29 @@ def undefine(vm_):
     return dom.undefine() == 0
 
 
-def purge(vm_, dirs=False):
+def purge(vm_, dirs=False, removables=None):
     '''
     Recursively destroy and delete a virtual machine, pass True for dir's to
     also delete the directories containing the virtual machine disk images -
     USE WITH EXTREME CAUTION!
 
+    Pass removables=False to avoid deleting cdrom and floppy images. NOTE THAT
+    THE DEFAULT VALUE IS DANGEROUS AND WILL BE CHANGED IN THE FUTURE.
+
     CLI Example:
 
     .. code-block:: bash
 
-        salt '*' virt.purge <domain>
+        salt '*' virt.purge <domain> removables=False
     '''
     disks = get_disks(vm_)
+    if removables is None:
+        salt.utils.versions.warn_until(
+            'Sodium',
+            'The \'removables\' argument default value has been deprecated '
+            'and changed to False. Note that removable images will no longer '
+            'be purged unless explicitely setting it to True.'
+        )
```

Comment:
you should also add removeables=True here.  We still want it to maintain the current behavior until that behavior is deprecated in Sodium.  Once we reach the time to work on the Sodium release, we will just remove this whole block, and set the default value in the definition to be `removables=False,`

And the message should be something like `The 'removables argument default value is True, but will be changed to False by default in {version}.  Please set to True to maintain the current behavior in the future.`